stable/artifactory/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # JFrog Artifactory Chart Changelog
 All changes to this chart will be documented in this file.
 
-## [7.12.19] - Mar 27, 2019
+## [7.13.1] - Mar 27, 2019
```

Comment:
@danielezer Can you please fix order of versions?

Reply:
@jainishshah17 Done